src/app/component/allresults/allresults.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="col text-center mb-3" style="margin-top: 50px">
```

Comment:
Could you plese remove this inline css ?

Reply:
yes i removed the inline css